qa/common/perf_analyzer_input_data_json/int_data_optional.json

```diff
@@ -0,0 +1,10 @@
+{
+    "data" :
+        [
+        
+            {
+                "INPUT0" : [1]
+            }
```

Comment:
Can we add another line for `INPUT1` too?

```
{
  "INPUT1" : [1]
}
```

Reply:
It's supposed to be missing to show that PA doesn't error when missing an optional input.

Reply:
```
{
    "data" :
        [

            {
                "INPUT0" : [1]
            },
            {
                "INPUT1" : [1]
            }
       ]
}       
```
I mean something like above. Just to make sure if the optional input changes, everything still works properly.

Reply:
Added 👍